content/src/content/jcr_root/apps/core/wcm/components/page/v3/page/headlibs.html

```diff
@@ -26,5 +26,12 @@
     <sly data-sly-test="${clientLibCategoriesJsHead}" data-sly-call="${clientlib.js @ categories=clientLibCategoriesJsHead, async=page.isClientlibsAsync}"></sly>
     <sly data-sly-test="${clientLibCategories}" data-sly-call="${clientlib.css @ categories=clientLibCategories}"></sly>
     <link data-sly-test="${staticDesignPath}" href="${staticDesignPath}" rel="stylesheet" type="text/css" />
-    <sly data-sly-test="${page.data && page.dataLayerClientlibIncluded}" data-sly-call="${clientlib.js @ categories='core.wcm.components.commons.datalayer.v1', async=true}"></sly>
+    <sly data-sly-test="${!page.dataLayerSkipAcdlInclude}">
```

Comment:
We already have `isDataLayerClientlibIncluded` that serves the same purpose (to control if the ACDL scripts should be included).

Reply:
see Page.java explanation above